devel/pulp/devel/test_runner.py

```diff
@@ -60,9 +69,28 @@ def run_tests(packages, tests_all_platforms, tests_non_rhel5):
                      ','.join(packages)])
 
     # don't run the server or plugins tests in RHEL5.
+    flake8_exit_code = 0
     if sys.version_info >= (2, 6):
+        # make sure we test everything
         args.extend(tests_non_rhel5)
+
+        # Check the files for coding conventions
+        if run_flake8:
+            if not flake8_paths:
+                flake8_paths = ['.']
+
+            flake8_args = ['flake8', '--config', 'flake8.cfg', ]
+            if flake8_excludes:
+                joined_flake8_excludes = ','.join(flake8_excludes)
+                flake8_args.extend(['--excludes', joined_flake8_excludes])
+
+            for flake_path in flake8_paths:
+                flake8_command = copy.copy(flake8_args)
+                flake8_command.append(flake_path)
+                flake8_exit_code = (flake8_exit_code or subprocess.call(flake8_command))
```

Comment:
flake8 accepts multiple paths as arguments, so rather than calling it several times like this you could just append all the args to flake_args and run it once. This is how run-tests.py currently works.